shimmingtoolbox/optimizer/sequential.py

```diff
@@ -24,7 +24,7 @@ def sequential_zslice(unshimmed, coils, full_mask, z_slices, bounds=None):
     """
     z_slices.reshape(z_slices.size)
     currents = np.zeros((z_slices.size, coils.shape[3]))
-    optimizer = BasicLSQ(coils)
+    optimizer = Opt(coils)
```

Comment:
explicit code is usually preferred for readability, why not simply calling `LSQ_Optimizer`?

Reply:
That's fair, I'll change that

Reply:
Fixed